### PR TITLE
New resource svm peer

### LINKED ISSUE #51: [New Resource]: /svm/peers
### Description

ability to create SVM peers

### Requested Resource(s) and/or Data Source(s)

* netapp_ontap_svm_peers

### Potential Terraform Configuration

_No response_

### References

https://library.netapp.com/ecmdocs/ECMLP2885799/html/#/svm/svm_peer_create

### Would you like to implement a fix?

None

### PR DESCRIPTION
**Pull request by suhasbshekar**
New resource #51

### CODE REVIEW

File: .github/workflows/acc_test.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ jobs:
           TF_ACC_NETAPP_HOST: ${{ secrets.TF_ACC_NETAPP_HOST }}
           TF_ACC_NETAPP_HOST2: ${{ secrets.TF_ACC_NETAPP_HOST2 }}
           TF_ACC_NETAPP_HOST3: ${{ secrets.TF_ACC_NETAPP_HOST3 }}
+          TF_ACC_NETAPP_HOST4: ${{ secrets.TF_ACC_NETAPP_HOST4 }}
```
Comment (carchi8py) [inline .github/workflows/acc_test.yml:25]: What host 4?

File: docs/resources/svm_peers_resource.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "ONTAP: SVM Peers"
+subcategory: "SVM Peers"
```
Comment (carchi8py) [inline docs/resources/svm_peers_resource.md]: SVM

File: docs/resources/svm_peers_resource.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+### Related ONTAP commands
+* vserver peer create
+* vserver peer accept
+* vserver peer modify
+* vserver peer delete
+
+## Supported Platforms
+* On-perm ONTAP system 9.6 or higher
+
+~> **NOTE:** Amazon FSx for NetApp ONTAP is not supported
```
Comment (carchi8py) [inline docs/resources/svm_peers_resource.md]: you sure on this?

**Review by carchi8py — CHANGES_REQUESTED** [review]

File: docs/resources/svm_peers_resource.md (hunk 3 above)
Comment (suhasbshekar) [reply docs/resources/svm_peers_resource.md]: Nope, shall I remove it?, I thought its for most of the resources for now

File: .github/workflows/acc_test.yml (hunk 1 above)
Comment (suhasbshekar) [reply .github/workflows/acc_test.yml:25]: HOST3 not sure what's the IP, so created HOST4 which is 10.193.182.108

**Review by carchi8py — APPROVED** [review]
Need to fix the merge conflict.

**Pull request merged**